src/tests/test_cases/sections.py

```diff
@@ -340,13 +359,15 @@ def test_missing_args_static_method(a, x, y, _test, z=3):  # noqa: D213, D407
 @expect("D417: Missing argument descriptions in the docstring "
         "(argument(s) y are missing descriptions in "
         "'test_missing_numpy_args' docstring)")
+@expect("D207: Docstring is under-indented")
```

Comment:
This seems like a bug that getting warning here as this is a correct docstring.

Reply:
This is an unrelated bug - it should be fixed by https://github.com/PyCQA/pydocstyle/pull/472 which resolves the bug https://github.com/PyCQA/pydocstyle/issues/234